core.js

```diff
@@ -743,6 +743,9 @@ addInputHandler('enr_glus', function (input) {
     //input = input.replace(/\W/g, '');
     state.vars.current_step = 'enr_glus';
 
+    if(input.length != 13){
+        input = '0'+ input;
+    }
```

Comment:
@shejasabin I think this if block should come after checking for "99" as otherwise we'll never let the user exit with a 99 code.

Reply:
@shejasabin maybe include this if block inside the "else"

Reply:
I made the changes and created a PR. I was having issues with git pushing on this one